Packs/Legacy/Integrations/integration-CarbonBlackDefense.yml

```diff
@@ -1336,65 +1334,65 @@ script:
       required: true
       description: The alert id
     outputs:
-    - contextPath: CarbonBlackDefense.GetAlertDetails.DeviceInfo.DeviceType
+    - contextPath: CarbonBlackDefense.GetAlertDetails.DeviceInfo.DeviceInfo.DeviceType
```

Comment:
So it is `DeviceInfo.DeviceInfo` here?
double?

Reply:
@anara123 yes, it's there since forever in the original integration